src/wp-includes/rest-api/endpoints/class-wp-rest-block-types-controller.php

```diff
@@ -0,0 +1,549 @@
+			'parent'           => 'parent',
+			'provides_context' => 'provides_context',
+			'uses_context'     => 'uses_context',
+			'supports'         => 'supports',
+			'styles'           => 'styles',
+			'textdomain'       => 'textdomain',
+			'example'          => 'example',
+			'editor_script'    => 'editor_script',
+			'script'           => 'script',
+			'editor_style'     => 'editor_style',
+			'style'            => 'style',
+		);
+		foreach ( $extra_fields as $key => $extra_field ) {
+			if ( rest_is_field_included( $key, $fields ) ) {
+				if ( isset( $block_type->$extra_field ) ) {
+					$field = $block_type->$extra_field;
+				} elseif ( array_key_exists( 'default', $schema['properties'][ $key ] ) ) {
+					$field = $schema['properties'][ $key ]['default'];
+				} else {
+					$field = '';
+				}
+				$data[ $key ] = rest_sanitize_value_from_schema( $field, $schema['properties'][ $key ] );
+			}
+		}
+
+		if ( rest_is_field_included( 'styles', $fields ) ) {
+			$styles         = $this->style_registry->get_registered_styles_for_block( $block_type->name );
+			$styles         = array_values( $styles );
+			$data['styles'] = wp_parse_args( $styles, $data['styles'] );
+		}
+
+		$context = ! empty( $request['context'] ) ? $request['context'] : 'view';
+		$data    = $this->add_additional_fields_to_object( $data, $request );
+		$data    = $this->filter_response_by_context( $data, $context );
+
+		$response = rest_ensure_response( $data );
+
+		$response->add_links( $this->prepare_links( $block_type ) );
+
+		/**
+		 * Filters a block type returned from the REST API.
+		 *
+		 * Allows modification of the block type data right before it is returned.
+		 *
+		 * @since 5.5.0
+		 *
+		 * @param WP_REST_Response $response   The response object.
+		 * @param object           $block_type The original block type object.
+		 * @param WP_REST_Request  $request    Request used to generate the response.
+		 */
+		return apply_filters( 'rest_prepare_block_type', $response, $block_type, $request );
+	}
+
+	/**
+	 * Prepares links for the request.
+	 *
+	 * @since 5.5.0
+	 *
+	 * @param WP_Block_Type $block_type Block type data.
+	 * @return array Links for the given block type.
+	 */
+	protected function prepare_links( $block_type ) {
+		$pieces    = explode( '/', $block_type->name );
+		$namespace = $pieces[0];
+		$links     = array(
+			'collection' => array(
+				'href' => rest_url( sprintf( '%s/%s', $this->namespace, $this->rest_base ) ),
+			),
+			'self'       => array(
+				'href' => rest_url( sprintf( '%s/%s/%s', $this->namespace, $this->rest_base, $block_type->name ) ),
+			),
+			'up'         => array(
+				'href' => rest_url( sprintf( '%s/%s/%s', $this->namespace, $this->rest_base, $namespace ) ),
+			),
+		);
+
+		if ( $block_type->is_dynamic() ) {
+			$links['https://api.w.org/render-block']['href'] = add_query_arg( 'context', 'edit', rest_url( sprintf( '%s/%s/%s', 'wp/v2', 'block-renderer', $block_type->name ) ) );
```

Comment:
```suggestion
			$links['https://api.w.org/render-block'] = array(
				'href' => add_query_arg( 'context', 'edit', rest_url( sprintf( '%s/%s/%s', 'wp/v2', 'block-renderer', $block_type->name ) ) ),
			);
```

Reply:
Fixed in https://github.com/WordPress/wordpress-develop/pull/317/commits/30053466afbc62ad11b238ba515328bfaa59cfdf

Reply:
That is a different change and makes the `href` an array which isn't right. I'm saying this should be formatted like other links, https://github.com/WordPress/wordpress-develop/blob/master/src/wp-includes/rest-api/endpoints/class-wp-rest-posts-controller.php#L1928.

Reply:
Fixed